website/docs/intelligence-kit/brain/brain-v2.md

```diff
@@ -0,0 +1,194 @@
+---
+id: brain-v2
+title: Brain v2.0
+---
+# Overview
```

Comment:
Don't just start pages with `Overview`. The top header should reflect what the page is for if it were viewed without the sidebar


```suggestion
# Brain v2.0
```

Reply:
If you've done this anywhere else, please change it